packager/react-packager/src/Server/index.js

```diff
@@ -291,7 +291,8 @@ class Server {
       // Clear cached bundles in case user reloads
       this._clearBundles();
       this._hmrFileChangeListener(absPath, this._bundler.stat(absPath));
-      return;
+      // when HMR clients connected, also allow live reloading clients
+      // return;
```

Comment:
@martinbigio Thanks for your explain clearly. I think if HMR works fine and can replace Live Reload, we no longer need Live Reload. Yes, it seems an edge case.

Thanks again.